integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage40.java

```diff
@@ -122,115 +134,40 @@ public static void initAll(@Namespaces(4) List<String> namespaces) {
-    assertTrue(buildAppArchive(defaultAppParams()
-            .srcDirList(Collections.singletonList(MII_BASIC_APP_NAME))
-            .appName(MII_BASIC_APP_NAME)),
-        String.format("Failed to create app archive for %s", MII_BASIC_APP_NAME));
-
-    // copy app archive to models
-    assertDoesNotThrow(() -> Files.copy(
-        Paths.get(ARCHIVE_DIR, MII_BASIC_APP_NAME + ".zip"),
-        Paths.get(modelsPath1.toString(), MII_BASIC_APP_NAME + ".zip"),
-        StandardCopyOption.REPLACE_EXISTING));
-
-    // unzip WDT installation file into work dir
-    unzipWDTInstallationFile(multipleAIPath1.toString());
+    // image1 with model files for domain config, ds, app and wdt install files
+    createAuxiliaryImageWithDomainConfig(miiAuxiliaryImage1, "/auxiliary");
+    // image2 with model files for jms config
+    createAuxiliaryImageWithJmsConfig(miiAuxiliaryImage2, "/auxiliary");
 
-    // create image1 with model and wdt installation files
-    createAuxiliaryImage(multipleAIPath1.toString(),
-        Paths.get(RESOURCE_DIR, "auxiliaryimage", "Dockerfile").toString(), miiAuxiliaryImage1);
-
-    // push image1 to repo for multi node cluster
-    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
-      logger.info("docker push image {0} to registry {1}", miiAuxiliaryImage1, DOMAIN_IMAGES_REPO);
-      dockerLoginAndPushImageToRegistry(miiAuxiliaryImage1);
-    }
-
-    // create stage dir for second auxiliary image with image2
-    Path multipleAIPath2 = Paths.get(RESULTS_ROOT, this.getClass().getSimpleName(), "multipleauxiliaryimage2");
-    assertDoesNotThrow(() -> FileUtils.deleteDirectory(multipleAIPath2.toFile()),
-        "Delete directory failed");
-    assertDoesNotThrow(() -> Files.createDirectories(multipleAIPath2),
-        "Create directory failed");
+  }
 
-    // create models dir and copy model, archive files if any
-    Path modelsPath2 = Paths.get(multipleAIPath2.toString(), "models");
-    assertDoesNotThrow(() -> Files.createDirectories(modelsPath2));
-    Path multipleAIPathToFile2 =
-        Paths.get(RESULTS_ROOT, this.getClass().getSimpleName(), "multipleauxiliaryimage2/test.txt");
-    String content2 = "2";
-    assertDoesNotThrow(() -> Files.write(multipleAIPathToFile2, content2.getBytes()),
-        "Can't write to file " + multipleAIPathToFile2);
-    assertDoesNotThrow(() -> Files.copy(
-        Paths.get(MODEL_DIR, "/model.jms2.yaml"),
-        Paths.get(modelsPath2.toString(), "/model.jms2.yaml"),
-        StandardCopyOption.REPLACE_EXISTING));
 
-    // create image2 with model and wdt installation files
-    createAuxiliaryImage(multipleAIPath2.toString(),
-        Paths.get(RESOURCE_DIR, "auxiliaryimage", "Dockerfile").toString(), miiAuxiliaryImage2);
+  /**
+   * Create a domain using multiple auxiliary images. One auxiliary image containing the domain configuration and
+   * another auxiliary image with JMS system resource, verify the domain is running and JMS resource is added.
+   */
+  @Test
+  @Order(1)
```

Comment:
Only 2 methods are ordered. Can u merge these two if it need ordering.

Reply:
I can move this test which creates the domain to initAll and remove the dependency for the other two tests instead of  merging 3 different use cases into one.